### PR TITLE
fix grafana bug

### PR DESCRIPTION
**Pull request by w169q169**
bug 1:
in plugin simple json 1.4.1,the order of data and time has been changed, value of number will be behind  value of time.
image
bug 2:
if change interval according to hours, the value will be remained all the way.
image

### CODE REVIEW

**Review by Genius-pig — APPROVED** [review]

File: grafana/src/test/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImplTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    public void getInternal() {
+        BasicDaoImpl impl = new BasicDaoImpl(null);
+        ReflectionTestUtils.setField(impl, "isDownSampling", true);
+        ReflectionTestUtils.setField(impl, "interval", "1m");
+
+        String internal1 = impl.getInternal(0);
+        assert internal1.equals("");
+
+        String internal2 = impl.getInternal(3);
+        System.out.println("123");
```
Comment (qiaojialin) [inline grafana/src/test/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImplTest.java:49]: please remove the print

**Review by qiaojialin — COMMENTED** [review]
Welcome~

File: grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java [hunk 2]
```diff
@@ -133,31 +133,44 @@ public static void setTimestampRadioX(String timestampPrecision) {
...
         from * timestampRadioX, to * timestampRadioX);
     String columnName = "root." + s;
-    if (isDownSampling && (hours > 1)) {
-      if (hours < 30 * 24 && hours > 24) {
-        interval = "1h";
-      } else if (hours > 30 * 24) {
-        interval = "1d";
-      }
+
+    String internalLocal = getInternal(hours);
```
[the remark this answers is not in the document]
Comment (w169q169) [reply grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java:145]: The value of internal is global in the past version

[the remark this answers is not in the document]
Comment (qiaojialin) [reply grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java:145]: should the internal be interval?

Interval means a time range, do you mean  an internal interval...?

[the remark this answers is not in the document]
Comment (w169q169) [reply grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java]: already fixed.

**Comment by Genius-pig** [comment]
can you add some documents to specify which version of IoTDB use which version of simplejson plugin? for example, if you use iotdb0.11, you should use simplejson 1.4.1.

**Review by qiaojialin — APPROVED** [review]

**Comment by qiaojialin** [comment]
Thank you! Sorry for the late reply...   You could add my WeChat if convenient:qiaojialin1993

**Comment by qiaojialin** [comment]
#2137

**Pull request merged**